test/unit/data/entity.js

```diff
@@ -183,52 +183,34 @@ describe('extracting entities from submissions', () => {
         });
       });
 
-      it('should reject if request does not form valid entity', () => {
```

Comment:
Why remove this test? Is it redundant? For what it's worth, I see "Label empty or missing." elsewhere below, but not "ID empty or missing."

Reply:
There is a test `should throw errors when id is missing` for that, however I've added one for "uuid to be string"